js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/ButtonSave.js

```diff
@@ -0,0 +1,67 @@
+/* global arangoHelper, $ */
```

Comment:
please use typescript and `tsx` for React code. All new code should be typescript. If it's not possible to do typescript right now, note this for future, but do change the extension to `jsx`. As a general practice, don't use `.js` for code that includes `jsx`, this can cause issues with other bundlers in case we ever want to switch to faster builds

Reply:
Will be solved in separate ticket https://arangodb.atlassian.net/browse/FE-204